### DISCUSSION TITLE
Horizontal background gradients?

### DISCUSSION THREAD

**Discussion by thegatesdev** [polybar/polybar#3086] [Q&A] [answered]
Heyo,

The docs have this snippet:
![image](https://github.com/polybar/polybar/assets/69715898/a8947395-6d8c-4433-a872-e1b3c1be205a)

Is it possible to make horizontal gradients for the background color?
It is not stated anywhere in the documentation, I tried playing around a bit but did not find out how to do it.

Thanks!

**Comment by patrick96** [comment, marked as answer]
We only have vertical gradients. And even those probably don't always work great.
